ichnaea/service/search/schema.py

```diff
@@ -38,9 +38,16 @@ class WifisSchema(SequenceSchema):
     wifi = WifiSchema()
 
 
+class FallbackSchema(MappingSchema):
+
+    lacf = SchemaNode(Integer(), missing=None)
+    ipf = SchemaNode(Integer(), missing=None)
+
+
 class SearchSchema(MappingSchema):
 
     radio = SchemaNode(String(),
                        validator=OneOf(RADIO_STRINGS), missing=None)
     cell = CellsSchema(missing=())
     wifi = WifisSchema(missing=())
+    fallbacks = FallbackSchema(missing={})
```

Comment:
A class level missing dict here is dangerous. I don't want to think about what this actually means in terms of sharing mutable global state. I'd rather make this None and deal with the dict transform later in the code.